storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -405,6 +431,14 @@ static my_bool get_one_option(int optid,
         extra_http_headers.insert(parse_http_header(argument));
       }
       break;
+    case OPT_CURL_RETRIABLE_ERRORS:
+      if (argument != nullptr) {
+        CURLcode code = static_cast<CURLcode>(atoi(argument));
```

Comment:
@altmannmarcelo Did you forget to push a followup commit which addressed the change? I still see atoi(argument) and no parsing of comma separated values

Reply:
@satya-bodapati correct, I've not added a new commit yet. I've addressed all comments apart from refactoring of the callback code. I will add them all on a single new commit and ask for a re-review.

Reply:
Ok, thanks @altmannmarcelo